bastet_js/src/JsArray.re

```diff
@@ -113,7 +109,7 @@ module Traversable: TRAVERSABLE_F =
         ArrayLabels.fold_right(
           ~f=
             (acc, x) =>
-              A.pure((x, y) => Js.Array.concat([|x|], y)) <*> f(acc) <*> x,
+              A.pure((x, y) => Alt.alt([|x|], y)) <*> f(acc) <*> x,
```

Comment:
Could you do the same for `bastet/src/Array.re` and have it reference `Alt.alt`?